project/frontend/src/FindMatch/FindMatch.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+
+class PreferenceForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      cuisine: '',
+      distance: '',
+      dining_experience: '',
+      price_level: '',
+      latitude: '',
+      longitude: '',
+      open: true,
+    };
+    this.changeState = this.changeState.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  // Updates the state of the input element so it holds the chosen value.
+  changeState(event) {
+    const value =
+      event.target.name === 'open' ? event.target.checked : event.target.value;
+    this.setState({ [event.target.name]: value });
+  }
+
+  handleSubmit(event) {
+    alert('the cuisine you chose is: ' + this.state.cuisine);
+    event.preventDefault();
+  }
+
+  render() {
+    const cuisines = ['italian', 'mexican', 'indian'];
+    const distances = [1, 5, 10, 25];
+    const dining_experiences = {
+      meal_takeaway: 'Takeout',
+      meal_delivery: 'Delivery',
+      restaurant: 'Eat In',
+    };
+    const prices = ['Low', 'Medium', 'High'];
+    return (
+      <form onSubmit={this.handleSubmit}>
+        <label>
+          What cuisine?
+          <select
+            name='cuisine'
+            id='cuisine'
+            onChange={this.changeState}
+            value={this.state.cuisine}>
+            {cuisines.map((cuisine) => (
+              <option value={cuisine}>
+                {cuisine[0].toUpperCase() + cuisine.slice(1)}
+              </option>
```

Comment:
You can just do the proper capitalization in the array. I realized since we are just using this for text search, case doesn't matter so I would just capitalize the first letter of each word in 32

Reply:
just changed it!